### PR TITLE
Ofmcc 1625 transport funding

### PR DESCRIPTION
**Pull request by jenbeckett**
also has UI wording updates from ticket 1834

 https://eccbc.atlassian.net/browse/OFMCC-1834

### CODE REVIEW

File: frontend/src/components/supp-allowances/SupportNeedsProgrammingAllowance.vue [hunk 1]
```diff
@@ -1,33 +1,79 @@
...
-        </li>
-      </ul>
+      An Inclusion Policy is a requirement to apply for Support Needs Supplementary Funding Providers participating in the OFM Test must have an Inclusion Policy to receive this funding. You can
+      upload your policy in
+      <router-link to="/account-mgmt">Account Management.</router-link>
     </v-col>
   </v-row>
+  <br />
+  <v-row no-gutters class="mr-2 my-2">
+    <v-btn v-if="!showMore" color="transparent" flat @click="toggleDiv" class="link-btn">Read More</v-btn>
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/components/supp-allowances/SupportNeedsProgrammingAllowance.vue:27]: I agree, and I did ask about this,
image

Maybe I should have just told them I'm using a button, but they wanted it to look like a link...

[the remark this answers is not in the document]
Comment (weskubo-cgi) [reply frontend/src/components/supp-allowances/SupportNeedsProgrammingAllowance.vue:27]: My suggested change will still make it look like a link.

File: frontend/src/views/supp-allowances/SupplementaryFormView.vue [hunk 2]
```diff
@@ -152,43 +168,72 @@ export default {
...
         supportFundingModel: [],
         supportOtherDescription: null,
         supplementaryApplicationId: undefined,
         supplementaryType: SUPPLEMENTARY_TYPES.SUPPORT,
+        id: uuid.v1(),
       }
 
       this.models = [{ ...this.findAndUpdateModel(suppApplications, indigenousProgrammingModel) }, { ...this.findAndUpdateModel(suppApplications, supportModel) }]
 
+      const transportApplications = suppApplications.filter((el) => el.supplementaryType == SUPPLEMENTARY_TYPES.TRANSPORT)
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/views/supp-allowances/SupplementaryFormView.vue:198]: done

File: frontend/src/views/supp-allowances/SupplementaryFormView.vue [hunk 3]
```diff
@@ -200,11 +245,53 @@ export default {
...
+      }
+
+      this.models.push(transportModel)
+    },
+    async deleteTransportModel(model) {
+      let index = this.models.indexOf(this.models.find((el) => model.supplementaryApplicationId && el.supplementaryApplicationId == model.supplementaryApplicationId))
+
+      //application exists in Dynamics, so we need to delete it first
+      if (index != -1) {
+        await ApplicationService.deleteSupplementaryApplication(model.supplementaryApplicationId)
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/views/supp-allowances/SupplementaryFormView.vue:274]: done

File: frontend/src/components/supp-allowances/TransportationAllowance.vue [hunk 4]
```diff
@@ -1,3 +1,192 @@
...
+    },
+  },
+  watch: {
+    transportModels: {
+      handler(value) {
+        this.models = cloneDeep(value)
+      },
+      deep: true,
+    },
+  },
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/components/supp-allowances/TransportationAllowance.vue:166]: done

File: frontend/src/views/supp-allowances/SupplementaryFormView.vue [hunk 5]
```diff
@@ -200,11 +245,53 @@ export default {
...
+      for (const model of this.models) {
+        if (!model.uploadedDocuments) {
+          continue
+        }
+        const foundDoc = model.uploadedDocuments.find((el) => el.documentId == documentId)
+        if (!foundDoc) {
+          continue
+        }
+        //don't wait for response, just delete it so the page can re render faster
+        DocumentService.deleteDocument(documentId)
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/views/supp-allowances/SupplementaryFormView.vue:291]: done

File: frontend/src/components/supp-allowances/TransportationAllowance.vue [hunk 6]
```diff
@@ -1,3 +1,187 @@
...
+      monthlyLeaseFormat: {
+        nullValue: '0.00',
+        min: 0,
+        decimal: '.',
+        separator: ',',
+        precision: 2,
+      },
+    }
+  },
+  computed: {},
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/components/supp-allowances/TransportationAllowance.vue]: done

File: frontend/src/services/documentService.js [hunk 7]
```diff
@@ -1,8 +1,8 @@
-import { isEmpty } from 'lodash'
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/services/documentService.js:1]: done

File: frontend/src/components/supp-allowances/SupportNeedsProgrammingAllowance.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/components/supp-allowances/SupportNeedsProgrammingAllowance.vue:27]: doen

File: frontend/src/components/supp-allowances/SupportNeedsProgrammingAllowance.vue [hunk 8]
```diff
@@ -96,5 +143,20 @@ export default {
       { label: 'Other', value: '4' },
     ]
   },
+  methods: {
+    toggleDiv() {
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/components/supp-allowances/SupportNeedsProgrammingAllowance.vue:147]: done

**Review by weskubo-cgi — APPROVED** [review]

**Pull request merged**